docs/_integration-manager/camel-routes/camel-route-local.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Camel Route Configuration for Local Storage
+nav_order: 1
+parent: File Folder Listener
+---
+# Camel Route Configuration for Local Storage
+
+## Overview
+
+The File Folder Listner can be configured with Camel route and its typically consists of a series of processing steps that are connected in a linear sequence.
+
+## Route Configuration
+
+Note that the File Folder Listener Service must be restarted for any configuration changes to take effect. Make sure you have already completed: File Folder Listener Authorization
+
+**routes-prototype.xml**
+(ProgramDataDirectory)/Actian/FileFolderListener/conf/routes-prototype.xml
+
+NOTE: Multiple routes (i.e. for AWS,Azure, GCP and Local) can be added into a single routes-prototype.xml
+
+Example:
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<routes xmlns="http://camel.apache.org/schema/spring">    
+    <route id="run-jobconfig-with-local-file" autoStartup="false">
+        <setProperty name="jobConfigId">
```

Comment:
Not sure, whether we need to specify these individual properties we have used in this configurations. 
Example - "jobConfigId" - What is this value? or anything ? Any suggestions?

This applicable for all type of route configurations.